tests/shared/robot.spec.js

```diff
@@ -131,16 +131,16 @@ describe('Shared: Robot', function() {
         data: {
           message: 'foo',
           code: 'MYERROR',
-          stacktrace: error.stack
+          stack: error.stack
         }
       });
     });
 
     it('should handle a string error', function() {
```

Comment:
> Oh, you've changed robot.printError back to {(Error|String)} :-/

I don't I touched that one at all, but I edited similar signatures all over the place, so I don't remember very much anymore :P

>  I wonder if it'd be worth keeping robot.printError as only accepting Error, and then using errors.createError in https://github.com/resin-io/etcher/blob/master/lib/cli/etcher.js#L124 to create a proper Error object if we only get a string back?

I guess we could, however I'd like to avoid `_.isError(error)` checks all over the place by making the function a bit smarter.

I edited the signatures to say we only accept Errors (to kinda enforce that), but the function will still handle other values, just in case.